plugins/keyboard-navigation/test/shortcuts_test.mocha.js

```diff
@@ -0,0 +1,416 @@
+/**
+ * @license
+ * Copyright 2021 Google LLC
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+const sinon = require('sinon');
+
+const Blockly = require('blockly/node');
+
+const {NavigationController} = require('../src/index');
+const {createNavigationWorkspace, createKeyDownEvent} =
+    require('./test_helper');
+
+suite('Shortcut Tests', function() {
+  /**
+   * Creates a test for not running keyDown events when the workspace is in read
+   * only mode.
+   * @param {Object} keyEvent Mocked key down event. Use createKeyDownEvent.
+   * @param {string=} optName An optional name for the test case.
+   */
+  function runReadOnlyTest(keyEvent, optName) {
```

Comment:
Could you swap these parameters around so they're the same as the other helper functions?
Also, since `optName` is always passed, you could drop the optional and just make it required. Up to you.

Reply:
done